packages/shared/src/utils/url/url.js

```diff
@@ -142,8 +142,9 @@ export const setUrlLanguage = lang => {
 
 export const getDerivComLink = (path = '') => {
     const host = 'https://deriv.com';
-    const lang = default_language?.toLowerCase();
-    const link_lang = lang === 'en' ? '' : `/${lang}`;
+    let lang = default_language?.toLowerCase();
+    if (lang && lang !== 'en') lang = `/${lang}`;
+    else lang = '';
```

Comment:
Any reason we're changing from `ternary` to this? Previous code does the same thing.

Reply:
We've discussed it [here](https://github.com/binary-com/deriv-app/pull/1142#discussion_r427099883)
Also please take a look at [this](https://paste.ubuntu.com/p/WmF6wHMQS4/)
Mahdi suggested using `??` as well but it does same thing as this code with extra conditions and initialization.